### PR TITLE
Update contributor workflow

### PR DESCRIPTION
**Pull request by casperklein**
# Description

This makes the "delete-old-branch" step optional.

Otherwise the action will fail, if the old branch does not exist (e.g. got already deleted by "auto-merge").

Fixes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/contributors.yml [hunk 1]
```diff
@@ -5,31 +5,41 @@ on:
...
-        PULL_REQUEST: 'master'
-        CONTRIBUTOR: '## Contributors'
-        COLUMN_PER_ROW: '6'
-        ACCESS_TOKEN: ${{secrets.GITHUB_TOKEN}}
-        IMG_WIDTH: '100'
-        FONT_SIZE: '14'
-        PATH: '/CONTRIBUTORS.md'
-        COMMIT_MESSAGE: 'docs(CONTRIBUTORS): update contributors'
-        AVATAR_SHAPE: 'round'
+      -
```
Comment (wernerfred) [inline .github/workflows/contributors.yml:23]: Can we avoid this syntax and put the name/uses in there

Comment (casperklein) [reply .github/workflows/contributors.yml:23]: Any reason for that or open for discussion? 😄 

Especially with many steps involved, this is much more readable. See our "default-on-push" with this style [here](https://gist.github.com/casperklein/7f839f94bf0070836737b111f04b52f6)

What do you think?

**Review by polarathene — APPROVED** [review]

**Review by georglauterbach — APPROVED** [review]

File: .github/workflows/contributors.yml (hunk 1 above)
Comment (casperklein) [reply .github/workflows/contributors.yml:23]: Sorry, I don't wanted to pass you over. I forgot that auto-merge was activated and the other two approved it pretty fast.

Comment (georglauterbach) [reply .github/workflows/contributors.yml:23]: I did not see it as well, sorry:)

Comment (casperklein) [reply .github/workflows/contributors.yml:23]: What's your opinion about the style? As you approved it, I think you are fine with it?
If there is con-sense, I can provide a further PR to adjust the other workflows as well.

Comment (wernerfred) [reply .github/workflows/contributors.yml:23]: No worries :)
I Just don't Like how this looks. Feels wrong for me and i Always remove those Lines If i have them Open in an Editor during Work. Better Option in my opinion would be to add empty Lines between the Tasks for better visibilty. 

No Matter which way we Go, i +1 your approach to Streamline the Syntax.

**Pull request merged**